src/utils/index.js

```diff
@@ -1,3 +1,19 @@
+import amplitude from "amplitude-js";
+
+export const setupAmplitude = () => {
+  if (typeof window === "undefined") {
+    return;
+  }
+
+  amplitude.getInstance().init("656f6573a88e63d3dc2ce32e33ef8284");
```

Comment:
I'd suggest moving this id to an environment variable in `.env` file. This way we can set up separate Amplitude instances for prod and labs environments.

Please see the following article: https://www.gatsbyjs.org/docs/environment-variables/